features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/messagecomposer/MessageComposerPresenter.kt

```diff
@@ -194,43 +194,46 @@ class MessageComposerPresenter @Inject constructor(
                         ongoingSendAttachmentJob.value == null
                     }
                 }
+                is MessageComposerEvents.Error -> {
+                    analyticsService.trackError(event.error)
+                }
             }
         }
 
         return MessageComposerState(
-            text = text.value,
+            composerState = composerState,
             isFullScreen = isFullScreen.value,
             hasFocus = hasFocus.value,
             mode = messageComposerContext.composerMode,
             showAttachmentSourcePicker = showAttachmentSourcePicker,
             canShareLocation = canShareLocation.value,
             canCreatePoll = canCreatePoll.value,
             attachmentsState = attachmentsState.value,
-            eventSink = ::handleEvents
+            eventSink = { handleEvents(it) }
```

Comment:
Workaround for a Kotlin reflection error when `toString()` is called in test assertion failures... Related: https://github.com/vector-im/element-x-android/pull/1206